Assets/Scripts/Augment/GunStats.cs

```diff
@@ -27,9 +27,13 @@ public enum FireModes
 
 
     //How many projectiles in a clip
-    public int magazineSize = 20;
-
-    public int Ammo = 20;
+    [SerializeField]
+    private ModifiableFloat magazine = new ModifiableFloat(20f);
+    public ModifiableFloat Magazine { get => magazine; }
```

Comment:
Please use a more understandable name than just magazine for the floating point magazine size that we adjust, and use => without the `get` keyword since there's no setter.

Reply:
Was following the style of the other stats, and I refuse to have different conventions in the same class. I can change ALL stats though, which I will instead.

Reply:
I also think naming it magazine is a good idea for the following reasons:

- The base body stats explicitely need an amount. While it *can* be a float, we actually want users to think it's an int anyway.
- Editing "Magazine" with multipliers in barrels and extensions doesn't feel counterintuitive considering that's how we already adjust things like "projectilespershot", which really also could be considered an int from name, but isn't.

Renaming it to something like magazineSize just seem superfluous in my opinion.

I am open to name suggestions though

Reply:
Thought of something like RawMagazineSize to indicate that this is _not_ the final value, but considering that this is _that_ one ScriptableObject class that we'd make a bloody mess by modifying... Yeah I get it.